apps/web/src/state/lottery/helpers.ts

```diff
@@ -94,7 +90,7 @@ export const fetchMultipleLotteries = async (lotteryIds: string[]): Promise<Lott
       } as const),
   )
   try {
-    const client = publicClient({ chainId: ChainId.BSC })
+    const client = viemClients[ChainId.BSC]
```

Comment:
Shouldn't we define `client` at the top of file? `viemClients[ChainId.BSC]` won't change between function calls, so I guess it's fine to define it only once?

Reply:
Sure it's just that's the way they did so I did not want to change it.
it's a matter of "do we want to improve their code as well ?" :p

Reply:
You're right, let's not move too much things.